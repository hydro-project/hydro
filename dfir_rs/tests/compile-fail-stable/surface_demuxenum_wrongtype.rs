../compile-fail/surface_demuxenum_wrongtype.rs