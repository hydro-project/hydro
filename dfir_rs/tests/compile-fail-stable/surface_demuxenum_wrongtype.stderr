error[E0271]: type mismatch resolving `<impl Pusherator<Item = Shape> as Pusherator>::Item == {integer}`
  --> tests/compile-fail-stable/surface_demuxenum_wrongtype.rs:13:41
   |
13 |         my_demux = source_iter(0..5) -> demux_enum::<Shape>();
   |                                         ^^^^^^^^^^ expected integer, found `Shape`
   |
note: required by a bound in `pivot_run_sg_1v1`
  --> tests/compile-fail-stable/surface_demuxenum_wrongtype.rs:13:41
   |
13 |         my_demux = source_iter(0..5) -> demux_enum::<Shape>();
   |                                         ^^^^^^^^^^ required by this bound in `pivot_run_sg_1v1`
